backend/app/tests/services/test_ai_model_selection.py

```
@@ -0,0 +1,123 @@
+"""
+测试AI模型选择功能
+验证不同AI任务使用不同模型的功能
+"""
+
+import pytest
+from unittest.mock import AsyncMock, patch, ANY, MagicMock
```

Flake8 (7.2.0)

[error] 'unittest.mock.ANY' imported but unused

(F401)